polyhost/services/brightness_diag.py

```
@@ -0,0 +1,214 @@
+                      min_val, max_val, pre_scale, gamma)
+    except Exception as e:
+        print(f"   get_irradiance_now() raised {type(e).__name__}: {e}")
+        print("   (the worker periodic swallows this -> brightness is NOT updated)\n")
+
+    # (b) Clear-sky path the way the code does it (naive local time) vs the
+    #     timezone-correct way — this exposes the tz bug.
+    _clearsky_timezone_probe(s)
+
+
+def _report_value(label, irr, min_val, max_val, pre_scale, gamma=1.0):
+    perceived, n = normalize(irr, min_val, max_val, pre_scale)
```

Ruff (0.15.17)

[warning] Unpacked variable `perceived` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)